sdk/src/main/java/com/google/cloud/dataflow/sdk/util/BigQueryTableRowIterator.java

```diff
@@ -121,8 +121,20 @@ public static BigQueryTableRowIterator fromTable(TableReference ref, Bigquery cl
 
   /**
    * Constructs a {@code BigQueryTableRowIterator} that reads from the results of executing the
-   * specified query in the specified project.
+   * specified query in the specified project with useLegacySql set to True.
+   *
+   * @deprecated use {@link #fromQuery(String, String, Bigquery, Boolean, Boolean)}.
    */
+  @Deprecated
+  public static BigQueryTableRowIterator fromQuery(
+      String query, String projectId, Bigquery client, @Nullable Boolean flattenResults) {
+    return fromQuery(query, projectId, client, flattenResults, Boolean.TRUE);
```

Comment:
why not `null` so the default is in only one place?

Also, add `/* useLegacySql */` to the constant either way.